source/fundamentals/databases-collections.txt

```diff
@@ -16,9 +16,12 @@ MongoDB organizes data into a hierachy of the following levels:
 - documents
 
 Databases are the top level of data organization in a MongoDB instance.
-Databases are organized into collections, which contain :manual:`documents
-<core/document/>`. Documents contain literal data such as strings,
-numbers, and dates as well as other (embedded) documents.
+Databases are organized into collections which contain **documents**.
+Documents are the lowest-level structure that holds your data. Documents
```

Comment:
I would just remove this sentence.  Scalar BSON values _within_ documents are the lowest-level structure that holds your data.

Reply:
Removed sentence.